inc/dropdown.class.php

```diff
@@ -3258,7 +3258,7 @@ public static function getDropdownFindNum($post, $json = true) {
          $filterHelpdesk = false;
       }
 
-      if ($filterHelpdesk) {
+      if ($filterHelpdesk && $DB->fieldExists($post['table'], 'is_helpdesk_visible')) {
```

Comment:
If there is no `is_helpdesk_visible` column, then this item should not be in `$CFG_GLPI["helpdesk_visible_types"])`.

Reply:
to be removed, will do a global commit for this and #7428